File: play/perf/perf.py
#!/usr/bin/python

import sys

N = 1024*1024*16

x = range(0,N);

s = 0.0

for i in range(0,N) :
    s = s + x[i]

print "summed "+str(N)+" doubles in Python, result "+str(s)

File: play/perf/perf.pl
#!/usr/bin/perl -w

$N=1024*1024*16;

@x = 0..$N;

$s = 0.0;

for ($i=0;$i<$N;$i++) {
    $s += $x[$i];
}

print "summed $N doubles in Perl, result $s\n";
